ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/PortedEndpoints.java

```diff
@@ -135,6 +136,8 @@ static void setupEndpoints(final Endpoints endpoints, final Server server,
         .produces(Versions.KSQL_V1_JSON)
         .produces(MediaType.APPLICATION_JSON)
         .handler(new PortedEndpoints(endpoints, server)::handleServerMetadataClusterIdRequest);
+    router.route(HttpMethod.GET, "/ws/query")
```

Comment:
nit: inconsistent use of `/ws/*` above (and in ServerVerticle) in contrast to `/ws/query` here.

Reply:
ack